examples/demo.js

```diff
@@ -270,23 +270,26 @@ window.onload = function () {
   * @param {string} imageDataURL - The data URL for the image.
   */
   function savePDF(imageDataURL) {
-    sequencer.getImageDimensions(imageDataURL, function(dimensions) {
-      // Get the dimensions of the image.
-      let pageWidth = dimensions.width;
-      let pageHeight = dimensions.height;
-
-      // Create a new pdf with the same dimensions as the image.
-      const pdf = new jsPDF({
-        orientation: pageHeight > pageWidth ? 'portrait' : 'landscape',
-        unit: 'px',
-        format: [pageHeight, pageWidth]
-      });
-
-      // Add the image to the pdf with dimensions equal to the internal dimensions of the page.
-      pdf.addImage(imageDataURL, 0, 0, pdf.internal.pageSize.getWidth(), pdf.internal.pageSize.getHeight());
-
-      // Save the pdf with the filename specified here:
-      pdf.save('index.pdf');
+    sequencer.getImageDimensions(imageDataURL, function(dimensions, isGIF) {
```

Comment:
Let's mark this for refactoring; we should be able to determine if it's a gif from the imageDataURL directly, with some string parsing. This could be done via a **generalized utility function called `isGIF()` and not required as a parameter**. As long as we clearly mark this it shouldn't be necessary in this PR. Thanks!